LearnositySDK/Utils/JsonObject.cs

```diff
@@ -23,6 +25,9 @@ public class JsonObject : ICloneable
 
         public JsonObject(bool isArray = false)
         {
+            // Set CurrentCulture to proper handle float data type in json
+            Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
+
```

Comment:
@gonzalozawa I'm not a fan of playing with threads. Especially that our SDK may be part of a much bigger environment where changing culture per thread may do a lot of bad things, e.g
- on this thread you're saying that user culture customisations are not taken on account (https://msdn.microsoft.com/en-us/library/system.globalization.cultureinfo.invariantculture(v=vs.110).aspx)
- on all other threads it will be different, which could cause wrong values being saved to DB, or sort of other issues

I would suggest to try this to change sth in this line: https://github.com/Learnosity/learnosity-sdk-asp.net/blob/07310f556ba4833c81ded9d761688ca7dd1d664a/LearnositySDK/Utils/JsonObject.cs#L634
```
sb.Append(this.df[key].ToString("R"));
```
Would it work (as per https://msdn.microsoft.com/en-us/library/dwhawy9k.aspx)?

I have limited testing availability here, so even if I accept code review I will assign you back to do functional and take it from there.